kcidb_io/schema/v1.py

```diff
@@ -561,7 +965,7 @@ def get_version(data):
             except KeyError:
                 minor = 0
             if isinstance(major, int) and isinstance(minor, int) and \
-               major >= 0 and minor >= 0:
+                    major >= 0 and minor >= 0:
```

Comment:
Pycharm reformat is responsible for this. I'll revert.

Reply:
It might be a good idea to do these, and to add a step to check the indents are good to the CI. However, that needs to be done in a separate commit, or a separate PR. Thanks!